ui/app/services/permissions.js

```diff
@@ -49,6 +49,12 @@ const API_PATHS = {
   settings: {
     customMessages: 'sys/config/ui/custom-messages',
   },
+  sync: {
```

Comment:
I'm still learning out this service works...but do you know why just having 
```js
sync: 'sys/sync/',
```
which ends in a `/` isn't sufficient here?

Reply:
nope! 🙃 This is what I pinged Chelsea about in the dev channel. The logic of comparison is backwards from what you'd think they be comparing. I'll ping you on that thread which has links, etc. But it's also something that I can more easily clarify in standup.